spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraTableRowReaderProvider.scala

```diff
@@ -29,13 +29,13 @@ trait CassandraTableRowReaderProvider[R] {
 
   protected def readConf: ReadConf
 
-  protected def fetchSize: Int = readConf.fetchSize
+  protected def fetchSize: Int = readConf.fetchSizeInRows
```

Comment:
We are using size in bytes, megabytes, rows in more places (like `BatchSize`). Why don't create something like trait `Size` with subclasses `SizeInBytes` and `SizeInRows`, and then use these types where appropriate?